tests/lean/misc/MutuallyRecursiveTraits.lean

```diff
@@ -0,0 +1,6 @@
+-- THIS FILE WAS AUTOMATICALLY GENERATED BY AENEAS
+-- [mutually_recursive_traits]
+import Base
+open Primitives
+
+namespace mutually_recursive_traits
```

Comment:
I don't understand : as you use `-abort-on-failure` this file shouldn't have been generated?...

Reply:
And yet it was, I just checked. I didn't dig any further, we can probably open an issue for that